lib/image.go

```diff
@@ -113,6 +114,13 @@ func ImageCopy(opts ImageCopyOpts) error {
 		}
 	}
 
+	// Set ForceManifestMIMEType
+	// Supported manifest type :- https://github.com/anuvu/image/blob/master/manifest/manifest.go#L35
```

Comment:
Can you link to the upstream containers/image here?

Reply:
Ping?

Reply:
Updated link, please review